easybuild/easyconfigs/a/Autotools/Autotools-20150119-intel-2015a.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'Bundle'
+
+name = 'Autotools'
+version = '20150119' # date of the most recent change
+
+homepage = 'http://autotools.io'
+description = """This bundle collect the standard GNU build tools: Autoconf, Automake and libtool"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
```

Comment:
Well, we haven't been very consistent either way, and there's no huge harm done?

This can go in for me, I don't have a big problem with it.

If anything, it will make module names in the default EasyBuild module naming scheme more consistent, rather than having `intel` for some, `GNU` for others, etc.

I consider that a good thing; it doesn't matter much that these tools are built on top of libraries that they don't need, since it's not different than how stuff is built against OS libraries, where 90% isn't used either (yet available).